docs/source/tutorial/read-write-data.rst

```diff
@@ -52,15 +52,18 @@ Resource.
 
 To authenticate, you can use the Inrupt `solid-client-authn
 <https://www.npmjs.com/package/@inrupt/solid-client-authn>`_ library.
-After authenticating with ``solid-client-authn``, |product| picks up
-the authenticated session and includes the user's credentials with each
-request.
+After authenticating, the ``Session`` object obtained from
+``solid-client-authn`` will give you a `fetch` function that you can pass to
+|product| to includes the user's credentials with a request.
```

Comment:
will give you a -> provides a ``fetch`` function. 
to includes -> to include
Maybe:

> After authentication, the ``Session`` objection obtained from ``solid-client-authn`` provides a ``fetch`` function. You can pass this function as an option to |product| to include the user's credentials with a request.

Reply:
Much better. 92f357b40447334014b70cdffe0e88a937a89400